### PR TITLE
Add 2 new tags, add interpunction, add extra note to readme to add public & published datasets

### PR DESCRIPTION
**Pull request by thomcsmits**

### CODE REVIEW

File: src/tags.json [hunk 1]
```diff
@@ -9,5 +9,7 @@
   "salmon": "Spatial data analysed with Salmon",
   "codex": "CODEX data",
   "slide-seq": "Slide-seq data",
-  "files": "Work with HuBMAP data files"
+  "files": "Work with HuBMAP data files",
+  "hra": "Human Reference Atlas data",
+  "sc": "Single cell analysis"
```
Comment (john-conroy) [inline src/tags.json:14]:
```suggestion
  "sc": "Single cell analysis",
  "R": "Template written in R"
```
Not sure what the right language is, but we should add the R tag as well.

**Review by john-conroy — COMMENTED** [review]
Thanks! Just a comment to add the R tag as well.

File: src/tags.json (hunk 1 above)
Comment (thomcsmits) [reply src/tags.json:14]: good point! I want to add a github workflow soon to check whether any new tags have been added and to prompt someone who adds a new tag to actually add it to this list

**Review by john-conroy — COMMENTED** [review]
Seems like there are some linting failures.

**Review by john-conroy — APPROVED** [review]

**Pull request merged**